include/odp/api/spec/traffic_mngr.h

```diff
@@ -1351,6 +1406,15 @@ typedef struct {
 	 * greater levels may be connected to the fan-in of tm_node's with
 	 * numerically smaller levels. */
 	uint8_t level;
+
+	/** The strict priority level assigned to packets going through this
+	 * fan-in node. In other words all packets going through node
+	 * will be assigned this priority overwriting previous priority.
+	 * in the range 0..max_priority. This field is only valid if packet
+	 * priority mode is ODP_TM_PKT_PRIO_MODE_OVERWRITE.
+	 * @see odp_tm_pkt_prio_mode_t
+	 */
+	uint8_t priority;
```

Comment:
As far as I can see, the max_priority configuration does not restrict the priority value but the number of distinct priorities a node has to be able to handle. So the allowed range should be from zero to ODP_TM_MAX_PRIORITIES - 1.

I would like to propose the following text:

```
New strict priority level assigned to packets going through this node
when packet priority mode is ODP_TM_PKT_PRIO_MODE_OVERWRITE. In other
packet priority modes this field is ignored. The new priority does not
affect packet processing in this node but in its destination node.

The value must be in the range 0..ODP_TM_MAX_PRIORITIES-1. Additionally,
the total number of possible priorities seen by the destination node
must not exceed the max priority configured for the destination node.

@see odp_tm_pkt_prio_mode_t, odp_tm_level_requirements_t::max_priority
```